packages/shared/src/components/tooltips/SimpleTooltip.tsx

```diff
@@ -20,6 +20,7 @@ export function SimpleTooltip({
   content,
   ...props
 }: SimpleTooltipProps): ReactElement {
+  let shouldShow = true;
```

Comment:
Can you add a comment here to explain that this resolves a focus issue?
Otherwise, we might get confused in the future.

Reply:
Good point, will do 👏